CMakeLists.txt

```diff
@@ -128,10 +128,14 @@ enable_testing()
 
 # Create helper method which adds a valgrind test with the given binary
```

Comment:
this branch has merge commits in it

Reply:
replaced by https://github.com/KDAB/cxx-qt/pull/716